resources/js/components/Elements/Slider.vue

```diff
@@ -42,6 +61,19 @@
                 this.showRight = (this.slider.offsetWidth + this.position) < this.slider.scrollWidth - 1
             },
 
+            setHover(value) {
+                this.hover = value
+            },
+            autoScroll() {
+                if (!this.hover) {
```

Comment:
Same here, using the https://vueuse.org/shared/useIntervalFn/ allows you to pause the interval function on hover and resume it when no longer hovering.
Saving from that logic having to be done in the autoScroll function